frontend/src/lib/components/IntervalFilter/IntervalFilter.tsx

```diff
@@ -1,19 +1,40 @@
 import { intervalFilterLogic } from './intervalFilterLogic'
 import { useActions, useValues } from 'kea'
 import { intervals } from 'lib/components/IntervalFilter/intervals'
-import { InsightType, IntervalType } from '~/types'
+import { BaseMathType, InsightType, IntervalType } from '~/types'
 import { insightLogic } from 'scenes/insights/insightLogic'
-import { LemonSelect } from '@posthog/lemon-ui'
+import { LemonSelect, lemonToast } from '@posthog/lemon-ui'
 
 interface InvertalFilterProps {
     view: InsightType
     disabled?: boolean
+    activeUsersMath?: BaseMathType.WeeklyActiveUsers | BaseMathType.MonthlyActiveUsers | null
 }
 
-export function IntervalFilter({ disabled }: InvertalFilterProps): JSX.Element {
+export function IntervalFilter({ disabled, activeUsersMath }: InvertalFilterProps): JSX.Element {
```

Comment:
`activeUsersMath` introduces some significant two-way coupling between `InsightDisplayConfig` and `IntervalFilter`. I suggest we make this a bit more abstract, by passing in a `Set` containing the math types used in the insight here, and doing ALL the checks in `IntervalFilter` (or `intervalFilterLogic`).
This way the parent of `IntervalFilter` does not need to be aware of which math types matter interval availability-wise – it only needs to know the insight's definition, which is trivial.

Reply:
Oh, actually realized another thing – `intervalFilterLogic` already has `filters`, so we don't need to touch `InsightDisplayConfig.tsx` at all